packages/server/src/script/fetchMGErrors.ts

```diff
@@ -0,0 +1,105 @@
+import { Logging } from '@google-cloud/logging'
+import { processEnvOrThrow } from '../common'
+import { MailgunHookBody, MailgunEventReason } from '../service/webhooks'
+import { promises as fs } from 'fs'
+import { MGFetchErrorsArgs } from './data/mgConditions.sample'
```

Comment:
I would make `MGFetchErrorsArgs` defined in a file that's checked into git and import it in `mgConditions.sample.ts`.

Reply:
Should've done this initially, now it's closely to what we've done in `resendSignupEmails.ts`